external-crates/move/crates/move-compiler/src/shared/known_attributes.rs

```diff
@@ -296,6 +303,34 @@ impl SyntaxAttribute {
     }
 }
 
+impl ErrorAttribute {
+    pub const ERROR: &'static str = "error";
+    pub const CODE: &'static str = "code";
+    pub const ERROR_CODE_MAX: u16 = 32767;
```

Comment:
Can you compute this off of the `ErrorBitset` library to avoid potential for drift?

Reply:
This is all going to be gone! So no need now :)